canvas_manage_course/views.py

```diff
@@ -77,7 +75,7 @@ def dashboard_course(request):
         'manage_people': {
             'visible': lti_permission_required_check(request, settings.CUSTOM_LTI_PERMISSIONS['manage_people'])},
         'manage_sections': {
-            'visible': _lti_role_allowed(request, manage_sections_lti_roles_permitted)},
+                        'visible': lti_permission_required_check(request, settings.CUSTOM_LTI_PERMISSIONS['manage_sections'])},
```

Comment:
Looks like we can remove the `_lti_role_allowed()` method at the bottom of this file too.